TestCloudService/TestCloudService.ccproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>2.7</ProductVersion>
+    <ProductVersion>2.9</ProductVersion>
```

Comment:
Are there any version-specific azure SDK references within the non-test lightblue components that we also need to update?